zio-schema-zio-test/shared/src/main/scala/zio/schema/PrimitiveGen.scala

```diff
@@ -0,0 +1,165 @@
+package zio.schema
+
+import java.math.{ BigDecimal, BigInteger }
+import java.time._
+
+import scala.jdk.CollectionConverters._
+
+import zio.random.Random
+import zio.test._
+
+object PrimitiveGen {
+
+  def apply(standardType: StandardType[_]): Gen[Sized with Random, _] =
+    standardType match {
+      case _: StandardType.UnitType.type       => Gen.unit
+      case _: StandardType.StringType.type     => Gen.anyString
+      case _: StandardType.BoolType.type       => Gen.boolean
+      case _: StandardType.ShortType.type      => Gen.anyShort
+      case _: StandardType.IntType.type        => Gen.anyInt
+      case _: StandardType.LongType.type       => Gen.anyLong
+      case _: StandardType.FloatType.type      => Gen.anyFloat
+      case _: StandardType.DoubleType.type     => Gen.anyDouble
+      case _: StandardType.BinaryType.type     => Gen.chunkOf(Gen.anyByte)
+      case _: StandardType.CharType.type       => Gen.anyASCIIChar
+      case _: StandardType.BigDecimalType.type => Gen.anyDouble.map(BigDecimal.valueOf)
+      case _: StandardType.BigIntegerType.type => Gen.anyLong.map(BigInteger.valueOf)
+      case _: StandardType.DayOfWeekType.type  => anyDayOfWeek
+      case _: StandardType.Month.type          => anyMonth
+      case _: StandardType.MonthDay.type       => anyMonthDay
+      case _: StandardType.Period.type         => anyPeriod
+      case _: StandardType.Year.type           => anyYear
+      case _: StandardType.YearMonth.type      => anyYearMonth
+      case _: StandardType.ZoneId.type         => anyZoneId
+      case _: StandardType.ZoneOffset.type     => anyZoneOffset
+      case _: StandardType.Duration            => anyDuration
+      case _: StandardType.Instant             => anyInstant
+      case _: StandardType.LocalDate           => anyLocalDate
+      case _: StandardType.LocalTime           => anyLocalTime
+      case _: StandardType.LocalDateTime       => anyLocalDateTime
+      case _: StandardType.OffsetTime          => anyOffsetTime
+      case _: StandardType.OffsetDateTime      => anyOffsetDateTime
+      case _: StandardType.ZonedDateTime       => anyZonedDateTime
+    }
+
+  val anyDayOfWeek: Gen[Random, DayOfWeek] =
```

Comment:
Many of these would be nice to contribute back to ZIO Test! /cc @adamgfraser

Reply:
Definitely!

@igrebenik Would you like to submit a PR for that or shall we open an issue?

Reply:
@adamgfraser Hi Adam! I will submit the PR!

Reply:
@igrebenik Awesome!